File: movie-recommendation-frontend/src/models/Video.ts

export class Video {
  iso_639_1: string;
  iso_3166_1: string;
  name: string;
  key: string;
  site: string;
  size: number;
  type: string;
  official: boolean;
  published_at: string;
  id: string;

  constructor() {
    this.iso_639_1 = '';
    this.iso_3166_1 = '';
    this.name = '';
    this.key = '';
    this.site = '';
    this.size = 0;
    this.type = '';
    this.official = false;
    this.published_at = '';
    this.id = '';
  }
}

File: movie-recommendation-backend-new/src/main/java/com/movierecommendation/backend/model/data/videos/Video.java
package com.movierecommendation.backend.model.data.videos;

import lombok.Data;
import lombok.Getter;
import lombok.Setter;

@Getter
@Setter
@Data
public class Video {
    private String iso_639_1;
    private String iso_3166_1;
    private String name;
    private String key;
    private String site;
    private int size;
    private String type;
    private boolean official;
    private String published_at;
    private String id;
}